aws/resource_aws_codepipeline_test.go

```diff
@@ -378,6 +378,40 @@ func TestAccAWSCodePipeline_multiregion_ConvertSingleRegion(t *testing.T) {
 	})
 }
 
+func TestAccAWSCodePipelineWithNamespace(t *testing.T) {
+	if os.Getenv("GITHUB_TOKEN") == "" {
+		t.Skip("Environment variable GITHUB_TOKEN is not set")
+	}
+
+	var p1 codepipeline.PipelineDeclaration
+	name := acctest.RandString(10)
+	resourceName := "aws_codepipeline.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t); testAccPreCheckAWSCodePipeline(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSCodePipelineDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSCodePipelineConfigWithNamespace(name),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSCodePipelineExists(resourceName, &p1),
+					testAccMatchResourceAttrRegionalARN(resourceName, "arn", "codepipeline", regexp.MustCompile(fmt.Sprintf("^arn:aws:codepipeline:[^:]+:[0-9]{12}:test-pipeline-%s", name))),
```

Comment:
This check is currently failing acceptance testing. This should fix it:

```
--- FAIL: TestAccAWSCodePipelineWithNamespace (34.15s)
    testing.go:669: Step 0 error: Check failed: Check 2/6 error: aws_codepipeline.test: Attribute 'arn' didn't match "arn:aws:codepipeline:us-west-2:*******:^arn:aws:codepipeline:[^:]+:[0-9]{12}:test-pipeline-v841hd0wxm", got "arn:aws:codepipeline:us-west-2:*******:test-pipeline-v841hd0wxm"
```

```suggestion
					testAccMatchResourceAttrRegionalARN(resourceName, "arn", "codepipeline", regexp.MustCompile(fmt.Sprintf("test-pipeline-%s", name))),
```